classes/author/maps/Schema.inc.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * @file classes/author/maps/Schema.inc.php
+ *
+ * Copyright (c) 2014-2020 Simon Fraser University
```

Comment:
Copyright dates (here and probably elsewhere)

Reply:
There are other `Copyright (c) 2014-2020` across the code - should I change everything? Should we create another issue and fix them all across projects?

Reply:
Now that it's looking like a Q1 2022 release, don't worry about them -- I'll make sure they all get standardized before the release.